app/renderer/api/materialize.js

```diff
@@ -53,5 +53,11 @@ module.exports = {
     },
     closeModals() {
         $('#config-modal').modal('close');
+    },
+    getLengthNavBarTabs() {
```

Comment:
you don't really need to do this. Navbar is the size of AppStatus.suites, I think is better and cleaner using that, instead of relying on jQuery

Reply:
I can not find suites in appstatus, would you confirm? so I inserted with jquery the length.

Reply:
Sorry, it is in TaskHandler

Reply:
I really enjoyed your tips. as I am new to the Vue, the manipulation of the state does not dominate very well :smile: